components/layout/block.rs

```diff
@@ -647,6 +647,14 @@ impl BlockFlow {
         &mut self.fragment
     }
 
+    pub fn stacking_relative_position(&self, coor: CoordinateSystem) -> Rect<Au> {
```

Comment:
We should be able to reuse this function in `block.rs`, and also in `inline.rs` (`InlineFlow::compute_absolute_position`). Also in `BlockFlow::transform_clip_to_coordinate_space` in `display_list_builder.rs`, and I may have missed others.

It's fine for me if you don't want to do that (is fine to file a followup for that).